### PR TITLE
Fix GPT3_Writer Section 1 Typo

### PR DESCRIPTION
**Pull request by the-eyecon**
Removed the extra "to also" and fixed the typo for "hyperparameters" in Section 1

### CODE REVIEW

File: GPT3_Writer/en/Section_1/Lesson_1_Write_your_first_prompt.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ Tell me a full story with a beginning, middle, and end titled "Farza and the Mag
 
 **Keep messing around with the temperature as you run these** **different prompts.** I like changing the temperature by like `0.1` a few times to test. Also, the same prompt at a different temperature can create wildly different results.
 
-So basically lesson here: you need to adjust more than just the prompt to get good results, you also to need to also adjust your hyperparamaters.
+So basically lesson here: you need to adjust more than just the prompt to get good results, you also to need adjust your hyperparameters.
```
[the remark this answers is not in the document]
Comment (the-eyecon) [reply GPT3_Writer/en/Section_1/Lesson_1_Write_your_first_prompt.md:231]: Oh ok,  How about like this:  "**So basically lesson here: you need to adjust more than just the prompt to get good results, you also need to adjust your hyperparameters.**" 

I think this is what the author actually meant to write. I got the "need" and "to" mixed up, and that's why it sounded weird the first time.

[the remark this answers is not in the document]
Comment (helloitsm3) [reply GPT3_Writer/en/Section_1/Lesson_1_Write_your_first_prompt.md:231]: sure. Change the text and I'll merge it

[the remark this answers is not in the document]
Comment (the-eyecon) [reply GPT3_Writer/en/Section_1/Lesson_1_Write_your_first_prompt.md]: Done , you can merge

**Comment by the-eyecon** [comment]
Done, you can merge now

**Review by helloitsm3 — APPROVED** [review]

**Pull request merged**